src/app/body/home/home.component.css

```diff
@@ -32,17 +32,38 @@
 .demoButton {
     background-color: var(--primary-color);
     color: #ffffff;
-    display: block;
+    display: inline-block;
     border-radius:50px;
-	text-align:center;
+    text-align:center;
     font-family: 'Plus Jakarta Sans', sans-serif;
     border: none !important;
 }
 
+.videoButton {
+    background-color: var(--primary-color);
+    color: #ffffff;
+    display: inline-block;
```

Comment:
what is the need for inline in display

Reply:
i need display to be "inline-block" in both .demoButton and .videoButton so that they stay in the same line. I tried with just "block" but that brings video button to the next line